NuRadioMC/EvtGen/NuRadioProposal.py

```diff
@@ -508,15 +511,18 @@ def __filter_secondaries(self,
                 energy = sec.energy * units.MeV
 
                 shower_type, code, name = self.__shower_properties(sec.type)
-
-                shower_inducing_prods.append(SecondaryProperties(distance, energy, shower_type, code, name))
+                
+                shower_inducing_prods.append(
+                    SecondaryProperties(distance, energy, shower_type, code, name, 
+                                        sec.parent_particle_energy * units.MeV))
```

Comment:
is this unit conversion save? This only works if the base unit it MeV. It is cleaner if you also divide by `pp_MeV` defined in this file.

Reply:
A few lines above the same is done with `sec.energy`. So if we want to do it we should be consistent and do it everywhere